lib/tasks/local_transactions.rake

```diff
@@ -1,27 +1,11 @@
 namespace :local_transactions do
 
-  desc "Download and import services, interactions and contacts"
-  task :fetch => :environment do
-    LocalAuthorityDataImporter.update_all
-  end
-
   desc "Download and import services, interactions and contacts, and subsequently remove 'ghost' interactions"
```

Comment:
Might need to remove ", interactions and contacts, and subsequently remove 'ghost' interactions"